nautobot/utilities/testing/mixins.py

```diff
@@ -189,6 +190,13 @@ def assertInstanceEqual(self, instance, data, exclude=None, api=False):
     def create_tags(cls, *names):
         """
         Create and return a Tag instance for each name given.
-        """
 
+        DEPRECATED: use fixtures instead if at all possible.
+        """
+        warnings.warn(
+            "create_tags() is deprecated and will be removed in a future Nautobot release. "
+            "Use fixtures such as nautobot/extras/fixtures.tags.json (provided in Nautobot 1.5 and later) instead.",
+            DeprecationWarning,
+            stacklevel=2,
```

Comment:
Should we leave the if Python version check here since we haven't confirmed dropping 3.7 for 1.5?

Reply:
The python version check is needed for `logging.warning(...,stacklevel=...)` - `warnings.warn(..., stacklevel=...)` works fine in 3.7.